container/go/cmd/sha256/sha256.go

```diff
@@ -0,0 +1,39 @@
+// Portable SHA256 tool.
```

Comment:
Please include a comment here about the sha256.py you mentioned below so we have a chance of finding it later.